apps/remix-ide/src/app/editor/contextView.js

```diff
@@ -28,7 +27,7 @@ class ContextView {
     this._view = null
     this._nodes = null
     this._current = null
-    this.sourceMappingDecoder = new SourceMappingDecoder()
+    this.sourceMappingDecoder = sourceMappingDecoder
```

Comment:
This is not an instantiation anymore?

Reply:
no. `SourceMappingDecoder` is not a class anymore